scio-core/src/main/java/com/spotify/scio/transforms/FutureHandlers.java

```diff
@@ -17,10 +17,7 @@
 package com.spotify.scio.transforms;
 
 import com.google.common.collect.Iterables;
-import com.google.common.util.concurrent.FutureCallback;
-import com.google.common.util.concurrent.Futures;
-import com.google.common.util.concurrent.ListenableFuture;
-import com.google.common.util.concurrent.MoreExecutors;
+import com.google.common.util.concurrent.*;
```

Comment:
is the wildcard import intended?

Reply:
IntelliJ did it but we don't enforce Java style so I didn't bother to fix.